media/css/hubs/_masthead.scss

```diff
@@ -146,14 +138,51 @@
             }
         }
     }
-
-    .fx-privacy-link {
-        display: none;
-    }
 }
 
 .other #nav-download-firefox,
 .oldwin #nav-download-firefox,
 .oldmac #nav-download-firefox {
     display: none;
 }
+
+html[dir="rtl"] {
+    #masthead {
+
+        .masthead-logo {
+            @media #{$mq-tablet} {
```

Comment:
Odd nesting, though it still compiles as expected. Should probably put the selector inside the media query just for cleanliness.